AWSAuthSDK/Sources/AWSMobileClient/AWSMobileClientExtensions.swift

```diff
@@ -118,6 +107,23 @@ extension AWSMobileClient: AWSIdentityProviderManager {
         let task = AWSTask.init(result: dict as NSDictionary)
         return task
     }
+    
+    func setLoginMapTaskUsing(tokens: Tokens?, error: Error?, to task: AWSTaskCompletionSource<NSDictionary>) {
+        if let tokens = tokens {
```

Comment:
nit: guard/let instead of nested if. Also, is it acceptable to `task.set(error: nil)`? since `error` is an optional arg?

Reply:
- Changed to guard/let. 
- Added check for error = nil